src/components/ui/graphElements.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+
+interface Value<_> {
+  children: ReactNode;
```

Comment:
This interface's name has little meaning, please make them more meaningful, and as for the best practices you should name your interfaces using PascalCase. Also I can't see a point on using Generics here


```suggestion
interface GraphProps {
```